plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/HumanTaskProjectWizard.java

```diff
@@ -137,12 +155,12 @@ private void doFinish(String containerName, String fileName, IProgressMonitor mo
         }
         IContainer container = (IContainer) resource;
         final IFile file = container.getFile(new Path(fileName));
-        final IFile wsdlfile = container.getFile(new Path("ApproveClaimTask.wsdl"));
-        final IFile cbWsdlfile = container.getFile(new Path("ApproveClaimCBTask.wsdl"));
-        final IFile htconfigfile = container.getFile(new Path("htconfig.xml"));
+        final IFile wsdlfile = container.getFile(new Path(taskName + "Task.wsdl"));
```

Comment:
Fixed in https://github.com/wso2/devstudio-tooling-bps/commit/1b5789567f9361a4df08ca65bf24fd1b1f132751